test/fixtures/dataElement/SimpleAbstract.txt

```diff
@@ -0,0 +1,10 @@
+Grammar:     DataElement 4.0
+Namespace:   shr.test
+Description: "The SHR test namespace"
+
+CodeSystem:     FOO = http://foo.org
+
+Abstract EntryElement:   Simple
```

Comment:
As noted in my review of the grammar PR, I'm not sure it makes sense to support abstract entries at all.